pkg/cmd/create/install_dependencies.go

```diff
@@ -37,19 +38,10 @@ var (
 		"kubectl",
 		"gcloud",
 		"helm",
-		"ibmcloud",
 		"tiller",
 		"helm3",
-		"kops",
-		"kvm",
-		"kvm2",
 		"ksync",
 		"oc",
-		"virtualbox",
-		"xhyve",
-		"hyperv",
-		"terraform",
```

Comment:
actually, I have second thoughts around this now. Right now it is unused, but there is a chance we utilize for the distribution. We might need to discuss first how we want to go about the distro.

Reply:
Lets drop it for now